en/config/actuators.md

```diff
@@ -1,96 +1,235 @@
 
+#### Motor Geometry: VTOL Tiltrotor
 
-### Bidirectional Motors
+The motor geometry for a [Generic Quadplane VTOL Tiltrotor](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_generic_quadplane_vtol_tiltrotor) is shown below (the approach for configuring other [VTOL tiltrotors](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_generic_quadplane_vtol_tiltrotor) will be similar).
 
-Some vehicles may use bidirectional motors (i.e. motor spins in direction 1 for lower output range and in direction 2 for the upper half).
+![Geometry motor: tiltrotor vtol](../../assets/config/actuators/qgc_geometry_tiltrotor_motors.png)
+
+- `Tilted by`: The associated servo used for tilting the motor.
+  The properties of this servo are defined in the [Motor Tilt Servo Geometry](#motor-tilt-servo-geometry).
+
+
+#### Motor Geometry: Standard VTOL
+
+The motor geometry for a [Generic Standard Quadplane VTOL Tiltrotor](../airframes/airframe_reference.md#vtol_standard_vtol_generic_quadplane_vtol) is shown below (the approach for configuring other "Standard VTOL" will be similar).
+
+![Geometry motor: standard vtol](../../assets/config/actuators/qgc_geometry_standard_vtol_motors.png)
+
+Motors have most of the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+There is an additional field to indicate the direction in which the motor moves the vehicle (for a standard VTOL, the hover motors are usually set "upwards" and the pusher motor is set to "forwards").
+
+- `Axis`: One of `Upwards`, `Downwards`, `Forwards`, `Backwards`, `Leftwards`, `Rightwards`, `Custom`
+  - If `Custom` is selected, then the UI displays three additional fields for setting the motor orientation.
+
+#### Motor Geometry: Other Vehicles
+
+Other vehicle types will define an appropriate motor geometry for their frame type.
+Once again these motors will generally have the same kinds of properties as shown above.
+
+For example, a fixed-wing vehicle may just have a single pusher moter, while a rover with differential steering will have a motor for throttle and for steering.
+
+#### Bidirectional Motors
+
+Some vehicles may use bidirectional motors (i.e. motor supports spinning in both directions)
 For example, ground vehicles that want to move forwards and backwards, or VTOL vehicles that have pusher motors that go in either direction.
 
 If bidiectional motors are used, make sure to select the **Reversible** checkbox for those motors (the checkbox is displayed as an "advanced" option).
 
-![Reversible](../../assets/config/actuators/qgc_geometry_reversible.png)
+![Reversible](../../assets/config/actuators/qgc_geometry_reversible_marked.png)
 
 Note that you will need to also ensure that the ESC associated with bidirectional motors is configured appropriately (e.g. 3D mode enabled for DShot ESCs, which can be achieved via [DShot commands](../peripherals/dshot.md#commands)).
 
 
+### Control Surfaces Geometry
```

Comment:
@bkueng @junwoo091400 I have updated this control surfaces geometry section. It needs more input from @bkueng now.

Why? All the stuff about how control surfaces work doesn't tell people how to set the values here. 

@bkueng around about line 163 we have this:
```
- `Roll scale`: Roll scale (normalised: -1 to 1)
- `Pitch scale`: Pitch scale (normalised: -1 to 1)
- `Yaw scale`: Yaw scale (normalised: -1 to 1)
- `Trim`: ?
- `Skew`: ?
```
What should we put for each of these? I.e. what do values in the range correspond to? What is the input, and what is the output if (say) the value is 0.7?

Is the "Control Surface Deflection Convention" useful for explaining the above? Left in place in case it is!

Reply:
@sfuhrer I leave this up to you

Reply:
@sfuhrer Can you comment on this whole section on control surfaces?
In particular, as I asked in ^^^
- What should we put for each of the fields? I.e. what do values in the range correspond to? What is the input, and what is the output if (say) the value is 0.7?
  - In other words, consider someone with a VTOL or FW frame that has an elevon (say). How do they decide what number to put in here?
- How do we describe trim? 
  - I "think" it it is like an offset to the default expected position. So you have a servo where you'd liked the lowest input value to be at the default position. However the unpowered servo is actually a bit less than this. So you're saying that 0 is actually at 0.1 say and that code should automatically account for this when setting values.
  - But I could be completely wrong, and that still doesn't tell me what the scale is. So same question. I have a vehicle, what is this trim, and how do I set it.
- Ditto for skew?

Reply:
>What should we put for each of the fields? I.e. what do values in the range correspond to? What is the input, and what is the output if (say) the value is 0.7?

It's a normalized effectiveness of the actuator around the corresponding axis. The numbers that are entered there are directly put into the allocation matrix, that is then inverted to get from desired moments (normalized) to control signals. Increasing the scale will reduce the deflection of the control surfaces (as it gets inverted). 
This is very low level stuff that a normal user shouldn't care about, unless it is for the tuning of coupled control surfaces (like an elevon, that controls both pitch and roll). In that case we should mention what I wrote above (if you want more deflection of a certain actuator around a certain axis, you can reduce the scale). But really make sure that for most setups the default scales are all you need (they appear automatically when selecting the control surface type).

Trim: yes it's simply an offset that is added on this actuator, normally to have it centered without input.

>Ditto for skew?

I don't know, where is that used?

We can link the devsummit talk Beat and I will make on CA (once we have it), that will include a walk-through.